src/logger.ts

```diff
@@ -60,3 +66,120 @@ export function sendCrashResponse({
     callback();
   }
 }
+
+export function loggingHandlerAddExecutionContext() {
+  interceptStdoutWrite();
```

Comment:
Also, we weren't planning to enable the stdout/stderr interception quite yet, but I'm okay with it being gated behind an env var. cc @matthewrobertson for additional opinion